roadmap.md

```diff
@@ -90,11 +40,14 @@ to be an exhaustive list.
 * [MSBuild](https://github.com/microsoft/msbuild)
 * [Docker Images](https://github.com/dotnet/dotnet-docker)
 
-This roadmap is intended to communicate project priorities for evolving and
-extending the scope of .NET Core.
+This roadmap is intended to communicate project priorities for evolving and extending the scope of .NET Core.
 
 # Supported OS Versions
 
+## .NET Core 2.1 - Supported OS Versions
+
+Proposal: TBD
+
 ## .NET Core 2.0 - Supported OS Versions
 
 Proposal:
```

Comment:
I intentionally left 2.0 as it is now - we need to move it into 2.0 release-notes. And I've heard @richlander plans major surgery on the table.